client/src/routes/Sessions/components/SessionUnavailable.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {useTranslation} from 'react-i18next';
+
+import Gutters from '../../../common/components/Gutters/Gutters';
+import HalfModal from '../../../common/components/Modals/HalfModal';
+import {Spacer16, Spacer24} from '../../../common/components/Spacers/Spacer';
+import {Body16} from '../../../common/components/Typography/Body/Body';
+
+const SessionUnavailable = () => {
+  const {t} = useTranslation('Component.SessionUnavailable');
+
+  return (
+    <HalfModal>
+      <Spacer16 />
+      <Gutters>
+        <Spacer24 />
+        <Body16>{t('description')}</Body16>
+        <Spacer24 />
+      </Gutters>
+    </HalfModal>
+  );
+};
+
+export default SessionUnavailable;
```

Comment:
Should this be named `SessionUnavailableModal` to conform with the other ones maybe?

Reply:
I can rename, should we rename UpgradeAccount as well?

Reply:
Ah yes, please! Thanks :)